connection.go

```diff
@@ -455,14 +468,7 @@ func (b *ConnectionBuilder) BuildContext(ctx context.Context) (connection *Conne
 
 	// Create the HTTP client:
 	// #nosec 402
```

Comment:
You need to move this ^ line to the place where the `http.Transport` object is created.

Reply:
Fixed